CHANGELOG.md

```diff
@@ -77,6 +78,7 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
 - Fixed missing bodysearch information if victim was killed without leaving a trace caused by a weapon hit
 - Fixed "body_confirm" MSTACK noise by batching all the kills from a body into one message. (by @EntranceJew)
 - Fixed "body_confirm" message sending before corpse confirmation message.
+- Fixed round scoreboard tooltips not being wide enough for their strings. (by @EntranceJew)
```

Comment:
this needs to be in the `unreleased` section

Reply:
✔ automatic merge must have missed, fixed

Reply:
i don't see it fixed yet.

Might just be my github dying idk

Reply:
turns out making a commit works best after staging files 🙃